Cargo.toml

```diff
@@ -16,10 +16,10 @@ include = ["Cargo.toml", "src/**/*.rs", "README.md", "LICENSE"]
 [dependencies]
 bitvec = { version = "0.20.1", default-features = false, features = ["alloc"], optional = true }
 cfg-if = "1.0"
-scale-info-derive = { version = "1.0.0", path = "derive", default-features = false, optional = true }
+scale-info-derive = { version = "1.0.1", path = "derive", default-features = false, optional = true }
```

Comment:
We'll need #137 too before releasing and I think that'll mean `1.1`? Wdyt @jsdw? @ascjones ?

Reply:
We may have to make this `2.0` since the major bump of the requirement for `parity-scale-codec` makes it a breaking change.

Reply:
Scale 3 is also edition 2021 with an increased minimum Rust version, so I completely agree there.

Reply:
I seem to be unable to formulate a rule about how to deal with dependency updates, how a breaking change downstream translates. It seems to be a common problem. Then there's the matter of MSRV, which I don't think we're taking into consideration at all here.

Reply:
There is no real consensus in the Rust community either. With a complex project like Substrate, I would err on too much safety and more manual upgrades, and implement some helper tools to make the overview of outdated deps easier.